### PR TITLE
project: add '-u' and '-U' switches

### LINKED ISSUE #7: Allow updating specific template values
We can currently "update" a template by passing the "-f" switch to override it.

Problem is that it will discard all our progress.

Create a -u switch that updates the version=, checksum=, distfiles= and homepage= values.

### PR DESCRIPTION
**Pull request by Cogitri**
fixes #7

### CODE REVIEW

File: src/cli.yml [hunk 1]
```diff
@@ -16,6 +16,14 @@ args:
...
       value_name: crate/gem/perldist
       help: Explicitly sets what kind of template we want to generate
+  - update:
+      short: u
+      long: update
+      help: Check if a new version for the package is available and if so, update 'version'
+  - update_all:
+      short: U
+      long: UpdateAll
+      help: Same as 'update', but also update 'distfiles' and 'homepage'
```
Comment (Cogitri) [inline src/cli.yml]: Not really happy with how these two switches are called right now (especially the longs) and their help string

Comment (Cogitri) [reply src/cli.yml]: @maxice8

File: src/cli.yml [hunk 2]
```diff
@@ -16,6 +16,14 @@ args:
       long: tmpltype
       value_name: crate/gem/perldist
       help: Explicitly sets what kind of template we want to generate
+  - update:
+      short: u
+      long: update
+      help: Check if a new version for the package is available and if so, update 'version'
```
Comment (maxice8) [inline src/cli.yml]: Check if a new version is available and update version and checksum fields

File: src/cli.yml (hunk 1 above)
Comment (maxice8) [reply src/cli.yml]: looks fine here

File: src/cli.yml [hunk 3]
```diff
@@ -16,6 +16,14 @@ args:
...
+  - update_all:
+      short: U
+      long: UpdateAll
```
Comment (maxice8) [inline src/cli.yml]: Full Update

Comment (Cogitri) [reply src/cli.yml]: Can't contain spaces, I guess `fullupdate` should work?

**Pull request merged**